### PR TITLE
[UBS Tariffs ] - fix create receiving station error #4813 #4783 #4796

### PR DESCRIPTION
**Pull request by sergoliarnik**
1. Add option to create receiving station with numbers in the name
2. Add default station status when receiving station is created

### CODE REVIEW

File: service-api/src/main/java/greencity/dto/courier/AddingReceivingStationDto.java [hunk 1]
```diff
@@ -13,6 +13,6 @@
 @ToString
 public class AddingReceivingStationDto {
     @NotNull
-    @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z-'\\s.]{1,30}")
+    @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z0-9-'\\s.]{1,30}")
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/courier/AddingReceivingStationDto.java:16]: please add tests

File: service-api/src/main/java/greencity/dto/courier/ReceivingStationDto.java [hunk 2]
```diff
@@ -13,10 +14,12 @@
 public class ReceivingStationDto {
     @Min(1)
     private Long id;
-    @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z-'\\s.]{1,30}")
+    @Pattern(regexp = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z0-9-'\\s.]{1,30}")
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/courier/ReceivingStationDto.java:17]: same here

File: service/src/main/java/greencity/mapping/station/ReceivingStationDtoMapper.java [hunk 3]
```diff
@@ -25,6 +25,7 @@ protected ReceivingStationDto convert(ReceivingStation receivingStation) {
             .createDate(receivingStation.getCreateDate())
             .createdBy(receivingStation.getCreatedBy().getRecipientName() + " "
                 + receivingStation.getCreatedBy().getRecipientSurname())
+            .stationStatus(receivingStation.getStationStatus())
```
Comment (RomaMocherniuk) [inline service/src/main/java/greencity/mapping/station/ReceivingStationDtoMapper.java:28]: same here

File: service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java [hunk 4]
```diff
@@ -513,6 +514,7 @@ private ReceivingStation buildReceivingStation(AddingReceivingStationDto dto, Us
             .name(dto.getName())
             .createdBy(user)
             .createDate(LocalDate.now())
+            .stationStatus(StationStatus.ACTIVE)
```
Comment (RomaMocherniuk) [inline service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java:517]: same here

File: service-api/src/test/java/greencity/dto/courier/AddingReceivingStationDtoTest.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AddingReceivingStationDtoTest {
+    void nameRegex(String name, boolean validates) throws NoSuchFieldException {
```
Comment (RomaMocherniuk) [inline service-api/src/test/java/greencity/dto/courier/AddingReceivingStationDtoTest.java:11]: please use Parameterized Tests

File: service-api/src/test/java/greencity/dto/courier/ReceivingStationDtoTest.java [hunk 6]
```diff
@@ -0,0 +1,75 @@
+package greencity.dto.courier;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ReceivingStationDtoTest {
```
Comment (RomaMocherniuk) [inline service-api/src/test/java/greencity/dto/courier/ReceivingStationDtoTest.java:10]: same here

**Review by RomaMocherniuk — APPROVED** [review]

**Review by NKorzhik — APPROVED** [review]

**Review by JJJazl — APPROVED** [review]

**Review by OlegVat — APPROVED** [review]

**Pull request merged**